app/code/Magento/InventoryCatalog/Plugin/InventoryApi/SetDataToLegacyCatalogInventoryAtSourceItemsSavePlugin.php

```diff
@@ -60,19 +69,23 @@ public function __construct(
     public function afterExecute(SourceItemsSaveInterface $subject, $result, array $sourceItems)
     {
         foreach ($sourceItems as $sourceItem) {
-            if ($sourceItem->getSourceCode() !== $this->defaultSourceProvider->getCode()) {
-                continue;
+            try {
```

Comment:
Need to revert try... catch
Looks like hack for integration tests

Reply:
Fixed.